src/plugins/orangehrmBuzzPlugin/config/routes.yaml

```diff
@@ -43,6 +43,15 @@ apiv2_buzz_comment:
   requirements:
     shareId: '\d+'
 
+apiv2_buzz_likes_on_share:
+  path: /api/v2/buzz/shares/{shareId}/likes
```

Comment:
```suggestion
  path: /api/v2/buzz/share/{shareId}/like
```

Reply:
```yaml
  path: /api/v2/buzz/shares/{shareId}/likes
```
this is correct (which is in plural).
ref: https://restfulapi.net/resource-naming/

Reply:
since it's correct did not change.